### ISSUE TITLE
Side pane launchers (Gog, Epic Games, EA App, Ubisoft Connect, ...) should have dedicated (re)install/uninstall buttons

### ISSUE THREAD

**Issue by excited-bore** [lutris/lutris#5604]
![lutris_sidepane](https://github.com/user-attachments/assets/4c479a3c-0972-4993-aa00-33b1d292abc8)

There's no way to get a list of installed files for these launchers from the sidepane either.

**Comment by danieljohnson2** [comment]
These "sources" are part of Lutris, and you can only turn them off, which hides them.

Though some of them require you to install a separate launcher "game", which you would remove like you would any game.

What would have you an uninstall button actually *do*?

**Comment by excited-bore** [comment]
![epic_games](https://github.com/user-attachments/assets/e8b0ca68-6cc7-4c68-83be-e8140007782d)
TL;DR - Run, Reload and Disconnect which comes of as pretty unintuitive UI design and I would at least suggest a clear 'Uninstall' option even if it were the same as a nameswap with 'Disconnect')

I (personally) had problems with reinstalling a launcher (epic games store) from which the original files where deleted but the configuration remained (wine)
Since it kept trying to launch at the path of the old files i thought about reinstalling the launcher but I *thought* the option was different from the 'Disconnect' option because i associated it more with 'logging out'.
To be completely fair there were only two options and one of them worked, but it took me a while and I feel like it could have been avoided if it clearly stated 'Uninstall' because I wouldn't believe im not the only one that would get confused because of that.

**Comment by strycore** [comment]
We're absolutely not putting an uninstall button on the sidebar. It wouldn't even be consistent with GOG / Humble which offer DRM free games and no launcher or Steam which Lutris can't uninstall.

> which the original files where deleted but the configuration remained (wine)

That's a different issue. We're not making changes to the UI because you encountered this.
